html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/legend-align-text-align.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<title>legend align to text-align</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<style>
+ legend { width: 13em }
+</style>
+<fieldset><legend>foo bar abcdefghijklmnopqrstuvwxyz</legend></fieldset>
+<fieldset><legend align=left>foo bar abcdefghijklmnopqrstuvwxyz</legend></fieldset>
+<fieldset><legend align=center>foo bar abcdefghijklmnopqrstuvwxyz</legend></fieldset>
+<fieldset><legend align=right>foo bar abcdefghijklmnopqrstuvwxyz</legend></fieldset>
+<fieldset><legend align=justify>foo bar abcdefghijklmnopqrstuvwxyz</legend></fieldset>
+<script>
+function test_align(selectorTest, expectedAlign) {
+  const testElm = document.querySelector(selectorTest);
+  test(() => {
+    assert_equals(getComputedStyle(testElm).textAlign, expectedAlign);
+  }, selectorTest);
+}
+
+test_align('legend', 'start');
```

Comment:
Just `for (const legend of document.querySelectorAll('legend'))` instead of testing per-selector, maybe? Anyway not a big deal.

Reply:
Yeah, though the selector is also used for the test name, so I think I'll leave it as-is.